installer/templates/phx_assets/app.js

```diff
@@ -9,9 +9,8 @@ import "../css/app.scss"
 //
 // Import deps with the dep name or local files with a relative path, for example:
 //
-//     import {Socket} from "phoenix"
-//     import socket from "./socket"
-//
+// import {Socket} from "phoenix"
+// import socket from "./socket"
```

Comment:
we favor the `import {Socket} from "phoenix"` syntax. Can you update the guide only to make it clearer, but leave the import? Thanks!

Reply:
done